spec/models/course/assessment/assessment_ability_spec.rb

```diff
@@ -153,12 +153,11 @@ def get_text_response_answer_for(submission)
       it { is_expected.not_to be_able_to(:force_submit_assessment_submission, published_started_assessment) }
 
       # Course Assessment Submissions
-      it { is_expected.to be_able_to(:view_all_submissions, coursemate_attempting_submission) }
```

Comment:
Hmm any reason behind the modification of these three abilities?

Reply:
We dont need to redeclare view_all_submissions ability here since we already have the read ability..